### PR TITLE
netdiscover 0.9 (new formula)

### PR DESCRIPTION
**Pull request by rogeriopradoj**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/netdiscover.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+class Netdiscover < Formula
+  desc "Active/passive ARP reconnaissance tool"
+  homepage "https://github.com/netdiscover-scanner/netdiscover"
+  url "https://github.com/netdiscover-scanner/netdiscover/archive/refs/tags/0.9.tar.gz"
+  sha256 "dc436992cae0e61ad8d14e1fb4d2650ec9e754c0024010e15d341afb98e28f70"
+  license "GPL-3.0-only"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
```
Comment (carlocab) [inline Formula/netdiscover.rb:9]:
```suggestion
  depends_on "automake" => :build
  uses_from_macos "libpcap"
```
Seems like you need this.

File: Formula/netdiscover.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    help_output = pipe_output("#{sbin}/netdiscover -h").split("\n")
+
+    expect_line_1 = "Netdiscover #{version} [Active/passive ARP reconnaissance tool]"
+    expect_line_4 = "Usage: "
+    expect_line_4.concat("#{sbin}/netdiscover ")
+    expect_line_4.concat("[-i device] [-r range | -l file | -p] [-m file] [-F filter]")
+    expect_line_4.concat(" [-s time] [-c count] [-n node] [-dfPLNS]")
+
+    assert_match expect_line_1, help_output[0]
+    assert_match expect_line_4, help_output[3]
```
Comment (carlocab) [inline Formula/netdiscover.rb:29]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

**Comment by chenrui333** [comment]
ping @rogeriopradoj

**Comment by rogeriopradoj** [comment]
Thanks for all attention given but I'm going to close this PR. After several tests I can't make the application runs OK like in Linux.

Thanks again!

**Pull request closed without merging**